Bottom: 3919bc16ef132887486aa9af6d7f6e256186ddc4
Top:    ae67715850c1e4254aeeb6d7eba594746b80123d
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-11-19 22:53:38 +0000

Refresh of add-hidgc-parameter-to-various

---

diff --git a/src/draw.c b/src/draw.c
index 3325334..9b45887 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -1238,7 +1238,7 @@ DrawTextLowLevel (hidGC gc, TextType *Text, Coord min_line_width)
 	      newline.Point1.Y += Text->Y;
 	      newline.Point2.X += Text->X;
 	      newline.Point2.Y += Text->Y;
-	      _draw_line (Output.fgGC, &newline);
+	      _draw_line (gc, &newline);
 	    }
 
 	  /* move on to next cursor position */
